### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by acpaquette**
Minor fixes for various drivers going into a 1.0 release.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Can we get a changelog? Also, a breakdown of the fixes?

**Comment by acpaquette** [comment]
@Kelvinrr Added changelog entries the describe changes

File: ale/drivers/nh_drivers.py [hunk 1]
```diff
@@ -99,6 +99,10 @@ def sensor_name(self):
     def frame_chain(self):
         self._props['exact_ck_times'] = False
         return super().frame_chain
+    
+    @property
+    def sensor_model_version(self):
+        return 2
```
Comment (Kelvinrr) [inline ale/drivers/nh_drivers.py:105]: are there any existing tests for the cam model version stuff that we need to add or update?

Comment (acpaquette) [reply ale/drivers/nh_drivers.py:105]: I will double check

**Review by AustinSanders — DISMISSED** [review]

**Review by AustinSanders — APPROVED** [review]

**Pull request merged**